types/n3/index.d.ts

```diff
@@ -142,8 +142,8 @@ export interface ParserOptions {
 export type ParseCallback<Q extends BaseQuad = Quad> = (error: Error, quad: Q, prefixes: Prefixes) => void;
 
 export interface N3Parser<Q extends BaseQuad = Quad> {
-    parse(input: string): Q[];
-    parse(input: string, callback: ParseCallback<Q>): void;
+    parse(input: string | fs.ReadStream): Q[];
```

Comment:
Isn't `fs.ReadStream` too specific? Something like `EventEmitter` may be more appropriate.

Reply:
It’s a Readable